uroot/root.go

```diff
@@ -73,6 +79,13 @@ var (
 		// This is for uroot packages. Is this a good idea? I don't know.
 		{name: "/pkg", mode: os.FileMode(0777)},
 	}
+	devs = []dev{
+		// chicken and egg: these need to be there before you start. So, sadly,
+		// we will always need dev.cpio or something like it.
+		//{name: "/dev/null", mode: os.FileMode(0660) | 020000, magic: 0x0103},
+		//{name: "/dev/console", mode: os.FileMode(0660) | 020000, magic: 0x0501},
+		{name: "/dev/tty", mode: os.FileMode(0666) | 020000, magic: 0x0501},
```

Comment:
since this is not a os.FileMode at all, but a linux mode, I suggest to make mode just a uint32.